draft-ietf-quic-transport.md

```diff
@@ -1378,28 +1378,38 @@ A server MUST include the original_connection_id transport parameter
 ({{transport-parameter-definitions}}) if it sent a Retry packet to enable
 validation of the Retry, as described in {{packet-retry}}.
 
-
 ### Values of Transport Parameters for 0-RTT {#zerortt-parameters}
 
-A client that attempts to send 0-RTT data MUST remember the transport parameters
-used by the server.  The transport parameters that the server advertises during
-connection establishment apply to all connections that are resumed using the
-keying material established during that handshake.  Remembered transport
+The transport parameters that the server advertises during connection
+establishment generally apply to all connections that are resumed using
```

Comment:
"generally" is a little to squishy a word to use here.  I'd prefer to be crisp:

> The value of transport parameters from a connection are remembered and apply to any 0-RTT packets that are sent in subsequent connections that are resumed from that connection, except for those transport parameters that are explicitly excluded.  Remembered transport parameters apply to the new connection until the handshake completes and the client starts sending 1-RTT packets.  Once the handshake completes, the client uses the transport parameters established in the handshake.
>
> Any transport parameter that does not or cannot apply to 0-RTT packets does not need to be remembered.  This includes max_ack_delay, <list>.  Any transport parameter that applies to the connection as a whole does not need to be remembered.  This includes disable_migration, <list>.  Endpoints are not obligated to remember and use unknown transport parameters.

Then I think we can tabulate transport parameters and add a "0-RTT" column, indicating whether it applies to 0-RTT or not.

Reply:
I like this wording. I'll try go get to this in the next few days.

Reply:
I took the first paragraph wholesale. I think "applies to the connection as a whole" is a confusing construction (all params apply affect the whole connection). Let me know what you think of the rewording.